integration-test/spark3/src/test/scala/org/apache/uniffle/test/RMSparkSQLTest.scala

```diff
@@ -0,0 +1,152 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.uniffle.test
+
+import com.google.common.collect.Maps
+import org.apache.commons.lang3.StringUtils
+import org.apache.spark.SparkConf
+import org.apache.spark.shuffle.RssSparkConfig
+import org.apache.spark.sql.{Dataset, Row, SparkSession}
+//import org.apache.uniffle.common.config.RssBaseConf.RSS_KRYO_REGISTRATION_CLASSES;
+import org.apache.uniffle.common.rpc.ServerType
+import org.apache.uniffle.coordinator.CoordinatorConf
+import org.apache.uniffle.server.ShuffleServerConf
+import org.apache.uniffle.server.buffer.ShuffleBufferType
+import org.apache.uniffle.storage.util.StorageType
+import org.apache.uniffle.test.IntegrationTestBase._
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.{BeforeAll, Test}
+
+import java.io.{File, FileWriter, PrintWriter}
+import java.util
+import java.util.Map
+
+object RMSparkSQLTest {
```

Comment:
Aha! . How about renaming the `RemoteMergeSparkSQLTest` . RM looks Yarn ResourceManager

Reply:
This test is based on draft pr https://github.com/apache/spark/pull/50248.
For me, when I mention RM, the first thing that comes to my mind is Yarn ResourceManager. But I don't have any other good names.

Reply:
Or I will rename RM to RMS or RS. In fact, merge sort is implemented on the server side, but there is no combine, and remote sort is fine.
If so, the previous code and documentation need to be modified. Maybe a new PR is needed.